### DISCUSSION TITLE
How to up load QGIS project to qfield cloud Organisation Plan

### DISCUSSION THREAD

**Discussion by Bvpraagh** [opengisch/QField#3975] [Q&A]
HI,
 I usually upload my QGIS projects via qfield sync which would upload project to my free Beta account. I now have the Organization Plan (though it still says I have Pro next to name and a pro account as well).

I am now unsure of the upload process. How do I make sure my QGIS project goes directly to my organisation plan? 
Can I just use Qfield Sync and it automatically loads into the Organisation Plan? Can I delete the Pro Plan that seems to be there?
If I create a project inside the organization plan, how do I get my files created on QGIS into   that project?
Do I also need to add the other collaborator or should they automatically have access as they are a member of the organisation? 

Sorry, these may be basic questions but I had it working well and now I am confused about new process.

thanks for any help.

**Comment by ZsanettMed** [comment]
Hello, 

Answers to your question see below :) 

How do I make sure my QGIS project goes directly to my organisation plan?
-Once you transferred your projects to your organization (Project settings -> Transfer ownership of this project), QFieldSync will automatically save changes on your project there (QFieldSync shows the project owner)

Can I just use Qfield Sync and it automatically loads into the Organisation Plan? 
-once, the project is created in your organization, you can have access to it via QFieldSync. 

Can I delete the Pro Plan that seems to be there?
-yes, edit profile->billing->change->subscription

If I create a project inside the organization plan, how do I get my files created on QGIS into that project?
-Via QFieldSync like before

Do I also need to add the other collaborator or should they automatically have access as they are a member of the organisation?
-They will have access to the projects. You can find more information about it here: https://docs.qfield.org/reference/qfieldcloud/concepts/?h=collaborator#project-collaborators

I hope that I have answered your questions!

Kind regards, 
Zsanett

**Comment by martingeh** [comment]
This is a question - not a reply/answer to the OP.

Hi Zsanett, 
This is the same issue I'm still having.  Your reply via email to me yesterday hasn't helped at all.
I have an account that created and owns an organisation; in order to allow my collaborators access to use the project and add data, it has to be moved to the organisation (which has 5GB storage).  That's all fine so long as the project can be uploaded to the 'base' account that only has a 100MB storage limit.  My project has about 300MB of data, so not all the files are successfully uploaded to the personal account.  From the personal account, it is simple enough to transfer ownership of the project to the organisation, but the issue is there is only 100MB of space on the personal account  and so I can't upload to the project as it is incomplete.
As per your email, I have logged in via my personal account, created a project within my organisation in QFieldCloud via desktop; this doesn't allow me (or I still haven't found out or figured out how) to add data from my local directory so that when I sync the mobile device, the project is populated with my data and available within the organization.
Is this perhaps an operating system issue?  I'm using Mac and iOS, though I have also tried to get this to work via Windows.  The project ultimately is viewed via the QField App on iOS.

**Reply by ZsanettMed** [reply]
Hi Martin, 
you wrote me via email, that you have this project on QGIS. Is that correct or have I got that wrong?

**Comment by martingeh** [comment]
Hi ZsanettYes I replied/commented today on someone else’s post. I tried to follow your reply to me about getting my project into QField; I’ve been able to get projects up to my organisation but the problem is I have a project that is about 300MB in size. My account only has 100MB storage and I don’t really want to pay two subscriptions to get my project in to my organisation. I created a project in Qfieldcloud (in my organisation) today and tried to then sync a project I have created on my laptop - it just doesn’t work. And for whatever reason I can’t transfer a project using cable (might be an iOS/Mac issue?).Anyway, I have to postpone this project as I can’t get it uploaded to my organisation and tomorrow I have a 1300km drive ahead of me to get to another project site. Feel free to reply or direct me to other resources with the view to getting this project uploaded - I will be able see emails again when I arrive at my destination tomorrow. Regards,Martin GehrmannSent from my iPhoneOn 8 Mar 2023, at 15:35, ZsanettMed ***@***.***> wrote:﻿
Hi Martin,
you wrote me via email, that you have this project on QGIS. Is that correct or have I got that wrong?

**Reply by ZsanettMed** [reply]
Hi Martin, 
I would suggest we continue here, there might be other affected user.
First of all I would like to let you know that, in any case, we will manage to upload your project. I am here to support you. :) 
So, let´s see where we stay: 
You could create a new empty project in QFieldCloud, then you opened QGIS on your laptop and tried to upload your project with the plugin QFieldSync. But the upload wasn´t possible, right? Could you please share a screenshot or the message you get at this point?

Have a safe drive! 
Best, 
Zsanett

**Comment by martingeh** [comment]
Hi Zsanett

Thanks for your continued support - and patience!

The one thing I really enjoy about long drives, is that I have time to think, and today’s drive was no different.
I spent a lot of time today considering everything that you’ve suggested I do and after my arrival tonight, I revisited your emails and then step by step, tried everything you’ve said.

Creating a project in my organisation in QFieldCloud is simple enough,  but after creating a project, I didn’t synchronise that project back to my computer.  I have now done that, added all the information I need and have successfully uploaded that to QField Cloud and then, synchronised that to my mobile device.

My problem is me - not taking the time to consider other approaches to this matter, where instead I have taken each step recommended literally and then getting frustrated when the objective is not achieved.

So, thank you Zsanett for your knowledge, patience and understanding.  You have helped me immensely getting this project to where I need it.
Now to get on with more planning and work.

Have a great day!

Regards,

Martin




On 9 Mar 2023, at 04:34, ZsanettMed ***@***.***> wrote:


Hi Martin,
I would suggest we continue here, there might be other affected user.
First of all I would like to let you know that, in any case, we will manage to upload your project. I am here to support you. :)
So, let´s see where we stay:
You could create a new empty project in QFieldCloud, then you opened QGIS on your laptop and tried to upload your project with the plugin QFieldSync. But the upload wasn´t possible, right? Could you please share a screenshot or the message you get at this point?

Have a safe drive!
Best,
Zsanett
